setup.cfg

```diff
@@ -0,0 +1,12 @@
+[metadata]
+long_description = file: README.md
+
+
+[options]
+package_dir =
+  = src
+packages = find:
+python_requires = >=3.7
+
+[options.packages.find]
+where = src
```

Comment:
I see in setuptools's document we can use pyproject.toml to set up setuptools.
<https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html>
Are we going to merge this to pyproject.toml? Or are there any good reasons that we should keep this file?

Reply:
for the package, no, toml is the predicted future.
for testing with pytest, ya, setup files are still needed to test without installing